R/derive_vars_cat.R

```diff
@@ -0,0 +1,218 @@
+#' Derive Categorization Variables Like `AVALCATy` and `AVALCAyN`
+#' @param dataset
+#' `r roxygen_param_dataset(expected_vars = c("by_vars", "definition"))`
+#' @param definition List of expressions created by: `exprs()`.
+#' Must be in the format of a tribble.
```

Comment:
Is this still true - I just see it wrapped in `exprs()`?

Reply:
oh i see in the function that it converts to a tibble?  Do we need to say tibble-like format?  I was expecting to see `tibble()` but I just see `exprs()`

Reply:
Yes I meant to say that the format expected looks like you would define a tribble, but as an exprs. 
(Note that I say tribble, not tibble, i.e. 
~var1, ~var2,
val1, var2val1,
val2, var2val2
...
)

Reply:
I think I should be a bit more explicit in the description

Reply:
I am updating the description